ui/static/ui/js/static_assets.jsx

```diff
@@ -1,8 +1,15 @@
 define('static_assets', [
-  'react', 'jquery', 'lodash', 'utils'], function (React, $, _, Utils) {
+  'react', 'jquery', 'lodash', 'utils'
+  ], function (React, $, _, Utils) {
   'use strict';
 
   var StatusBox = Utils.StatusBox;
+  var InfiniteList = Utils.InfiniteList;
+
+  // Guess at a good value for the height of each element. Infinite requires
+  // an elementHeight for each item. If we wanted to do a per item elementHeight
+  // we could make this a list instead.
+  var elementHeight = 50;
```

Comment:
I'm fine with this, but I'm wondering if we need to test on all the browsers if it has the same behavior (same for the other number below)